README.md

```diff
@@ -9,9 +9,29 @@ Also included:
   + APIs for packet data over radio [README_data](README_data.md)
   + An STM32 embedded version of FreeDV 1600/700D/700E for the [SM1000](stm32/README.md)
 
-## Older code
+## Old Code and Deprecated Features
 
-In July 2023 this repo was refactored, older code can be found in https://github.com/drowe67/codec2-dev
+In July 2023 this repo was refactored, older code can be found in https://github.com/drowe67/codec2-dev. 
+
+We are currently conducting a major re-development of Codec 2, new speech coding, modems, and FreeDV modes are under active development.  We have limited resources available for maintenance or development on features that may soon be replaced (with the exception of major bugs).  We'd rather put our efforts into new features! Modes and Features we are not actively maintaining at present, and likely to be superseded in the near future include:
+1. FreeDV 1600, 700C, 700D, 700E, 2020.
+1. Codec 2 modes with the exception of Codec 2 3200 for M17.
+1. Our fork of LPCNet.
+1. The stm32/SM1000 firmware (paused until new modes available).
```

Comment:
SM1000 is probably not a mode or feature in itself.

```suggestion

Development of the STM32/SM1000 firmware is also paused until new FreeDV modes become available.
```

Reply:
It's the same point as above that we want to capture with this section - we don't want to be diverting resources into SM1000 firmware maintenance right now.  When we have something new and exciting (that's working will on say x86), we'll schedule an upgrade.

Reply:
I was mainly splitting out this into a separate category since the SM1000 not a Codec2 thing per se, just a client of it that also happens to require a significant number of changes to reduce requirements enough for it to work. :) Feel free to word as required.